engine/schema/src/main/java/com/cloud/storage/dao/VMTemplateZoneDaoImpl.java

```diff
@@ -55,22 +55,22 @@ public VMTemplateZoneDaoImpl() {
     public List<VMTemplateZoneVO> listByZoneId(long id) {
         SearchCriteria<VMTemplateZoneVO> sc = ZoneSearch.create();
         sc.setParameters("zone_id", id);
-        return listIncludingRemovedBy(sc);
+        return listBy(sc);
     }
 
     @Override
     public List<VMTemplateZoneVO> listByTemplateId(long templateId) {
         SearchCriteria<VMTemplateZoneVO> sc = TemplateSearch.create();
         sc.setParameters("template_id", templateId);
-        return listIncludingRemovedBy(sc);
+        return listBy(sc);
     }
 
     @Override
     public VMTemplateZoneVO findByZoneTemplate(long zoneId, long templateId) {
```

Comment:
This is the one that we might need to have some eyes on to ensure it will be ok.